### PR TITLE
JPERF-444: Reproduce slow `apt-get` issue

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

File: src/test/kotlin/com/atlassian/performance/tools/sshubuntu/api/ssh/SshUbuntuTest.kt [hunk 1]
```diff
@@ -20,6 +22,16 @@ class SshUbuntuTest {
...
     }
 
+    @Test
+    fun shouldInstallPackagesFast() {
+        val start = Instant.now()
+        val result = execute("apt-get update && export DEBIAN_FRONTEND=noninteractive; apt-get install gnupg2 -y -qq")
+        val duration = Duration.between(start, Instant.now())
+
+        assertThat(result.isSuccessful()).isTrue()
+        assertThat(duration).isLessThan(Duration.ofSeconds(15))
```
Comment (dagguh) [inline src/test/kotlin/com/atlassian/performance/tools/sshubuntu/api/ssh/SshUbuntuTest.kt:32]: Love it

**Review by dagguh — DISMISSED** [review]

File: docker/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,4 @@
 FROM rastasheep/ubuntu-sshd:18.04
 
+RUN echo "deb mirror://mirrors.ubuntu.com/mirrors.txt xenial main restricted universe multiverse\n" > /etc/apt/sources.list
```
Comment (dagguh) [inline docker/Dockerfile:3]: I didn't think of that before 😲

**Review by dagguh — APPROVED** [review]

File: docker/Dockerfile [hunk 3]
```diff
@@ -1,3 +1,4 @@
 FROM rastasheep/ubuntu-sshd:18.04
```
Comment (wyrzyk) [inline docker/Dockerfile:2]: I'm considering adding `apt-get update` here. @dagguh WDYT?

Comment (dagguh) [reply docker/Dockerfile:2]: We should do it.
Gotchas: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

**Pull request merged**